source/mainnet/smart-contracts/tutorials/wCCD/wCCD-introduction.rst

```diff
@@ -0,0 +1,91 @@
+.. _wCCD-introduction:
+
+=====================================
+Understanding the wCCD token protocol
+=====================================
+
+The native currency on the Concordium blockchain is CCD. When other tokens are
+built on the Concordium blockchain, they often use the recommended ``CIS-2``
+token standard. This has the advantage that other dApps (decentralized apps)
+can rely on some basic rules on how to interact with the ``CIS-2``
+tokens and on some basic rules on how the apps can retrieve events and data from the ``CIS-2`` tokens.
+The native currency CCD has a special intention in the Concordium
+blockchain network and does not comply with the ``CIS-2`` token standard.
+
+.. note::
+
+    The `CIS-2 standard <https://proposals.concordium.software/CIS/cis-2.html>`_
+    can represent fungible and non-fungible tokens.
+    It combines the Ethereum ``ERC20`` and ``ERC721`` standards with some modifications.
+    The `CIS-2 library <https://github.com/Concordium/concordium-rust-smart-contracts/blob/main/concordium-cis2/src/lib.rs>`_
+    is meant to be imported by ``CIS-2`` tokens.
+    Please explore the four ``CIS-2`` token standard implementation examples in the Concordium
+    smart contract repo:
+
+    - `wccd <https://github.com/Concordium/concordium-rust-smart-contracts/blob/main/examples/cis2-wccd/src/lib.rs>`_
+    - `upgradable wccd <https://github.com/Concordium/concordium-rust-smart-contracts/pull/128>`_
+    - `nft <https://github.com/Concordium/concordium-rust-smart-contracts/blob/main/examples/cis2-nft/src/lib.rs>`_
+    - `multi <https://github.com/Concordium/concordium-rust-smart-contracts/blob/main/examples/cis2-multi/src/lib.rs>`_
+
+
+Implementing two interfaces (one for CCD and another for ``CIS-2`` tokens)
+within the same smart contract can be cumbersome for developers and adds
+complexity. We need a process that converts CCD into a token (named wCCD) that is ``CIS-2``
+compliant so dApps can interact with it easily. For example, decentralized
+exchanges depend on the wCCD token because the wCCD token allows you to trade
+your CCD for other ``CIS-2`` tokens seamlessly.
```

Comment:
re-wrote it to:
For example, decentralized exchanges depend on the wCCD token because the wCCD token can be traded with other ``CIS-2`` tokens.

Reply:
I think the problem was the connection to the previous sentence. I think this fixes it:

```For example, for decentralized
exchanges that depend on the wCCD token because the wCCD token allows you to trade
your CCD for other ``CIS-2`` tokens seamlessly.```